### PR TITLE
Component library version update and refactoring

### PR DESCRIPTION
**Pull request by kiwansim**
there's a lot of refactoring in this PR to use the component library icons as much as possible. other substantive changes made are:
- [sc-14520] update x/Twitter, facebook, and SMS icons
- [sc-13882] use RadioButton and TextInput components from library

### CODE REVIEW

File: client/src/components/UserTypeInput.tsx [hunk 1]
```diff
@@ -51,87 +50,78 @@ const UserTypeInputWithoutI18n = (props: UserTypeInputProps) => {
...
           onChange={handleRadioChange}
+          name="user-type"
         />
-        <label htmlFor={"user-type-input-other"}>{USER_TYPES.other}</label>
       </div>
       {activeRadio === USER_TYPES.other.toLocaleUpperCase("en") && (
-        <input
+        <TextInput
           autoFocus
+          labelText={USER_TYPES.other} // required field but hidden by css
```
[the remark this answers is not in the document]
Comment (austensen) [reply client/src/components/UserTypeInput.tsx:122]: oh yeah good thinking about the screen reader - this makes sense to me!

File: client/src/styles/PortfolioTable.scss [hunk 2]
```diff
@@ -76,16 +76,6 @@ $table-border-dark: 1px solid $dark;
         font-weight: 400;
       }
 
-      .arrow-icon {
-        cursor: pointer;
```
Comment (austensen) [inline client/src/styles/PortfolioTable.scss:80]: I think we should keep the `cursor: pointer;` for the arrows to it's clear you can click them

Comment (austensen) [reply client/src/styles/PortfolioTable.scss:80]: oh and probably the `margin-left: 0.3125rem;` (5px) as well to keep it the same. 
image

File: client/src/components/PortfolioFilters.tsx [hunk 3]
```diff
@@ -2,7 +2,8 @@ import { t, Trans, Plural } from "@lingui/macro";
 import { withI18n, withI18nProps } from "@lingui/react";
 import classnames from "classnames";
 import React from "react";
-import { CheckIcon, ChevronIcon, CloseIcon, InfoIcon } from "./Icons";
+import { InfoIcon } from "./Icons";
```
Comment (austensen) [inline client/src/components/PortfolioFilters.tsx:5]: I'll make a ticket to add this to JFCL

**Review by austensen — APPROVED** [review]
This all looks great! So nice to see all that css cut out for the component library

File: client/src/styles/PortfolioTable.scss (hunk 2 above)
Comment (kiwansim) [reply client/src/styles/PortfolioTable.scss:80]: oh good call, I don't think I realized these were the icons that the classname was referring to

**Pull request merged**